extensions/ql-vscode/package.json

```diff
@@ -1272,8 +1272,8 @@
     "test": "npm-run-all -p test:*",
     "test:unit": "jest --projects test",
     "test:view": "jest --projects src/view",
-    "integration": "node ./out/vscode-tests/run-integration-tests.js no-workspace,minimal-workspace",
-    "integration:no-workspace": "node ./out/vscode-tests/run-integration-tests.js no-workspace",
+    "integration": "npm-run-all integration:*",
+    "integration:no-workspace": "jest --config=out/vscode-tests/no-workspace/jest.config.js out/vscode-tests/no-workspace",
```

Comment:
How come we're not using `jest --projects`?

Reply:
I hoped that this would make the tests work on Windows, but it unfortunately didn't work. I've reverted to using `--projects` now.